app/components/component-docs/dataDisplay/CoreAvatar.docs.js

```diff
@@ -85,11 +90,9 @@ export default function CoreAvatarDocs() {
           >
             <CoreAvatar>W</CoreAvatar>
```

Comment:
Why do we change two letter words to single?

Reply:
Changed letter due to keep similar to MUI Avatar documentation.